templates/overrides/semantic/web/Views/Shared/_ValidationScriptsPartial.cshtml

```diff
@@ -0,0 +1,17 @@
+<environment names="Development">
+    <script src="~/lib/jquery-validation/dist/jquery.validate.js"></script>
+    <script src="~/js/semantic.validation.js"></script>
+    <script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.js"></script>
+</environment>
+<environment names="Staging,Production">
+    <script src="//ajax.aspnetcdn.com/ajax/jquery.validate/1.14.0/jquery.validate.min.js"
+            asp-fallback-src="~/lib/jquery-validation/dist/jquery.validate.min.js"
+            asp-fallback-test="window.jQuery && window.jQuery.validator">
+    
+    </script>
+    <script src="~/js/semantic.validation.js"></script>
+    <script src="//ajax.aspnetcdn.com/ajax/mvc/5.2.3/jquery.validate.unobtrusive.min.js"
```

Comment:
It has to be full link with https in templates:
https://github.com/OmniSharp/generator-aspnet/commit/73b82809c1694ad69b66532cb04eac7b1540c865
I know it will hurt developers when working locally, but that's requirement:
https://twitter.com/paul_irish/status/588502455530311680?lang=en